installer/driverInstall.js

```diff
@@ -113,14 +113,20 @@ var install_node_ibm_db = function(file_url) {
     var clidriverFound = true;
 
     if (process.env.IBM_DB_HOME) {
-      if (fs.existsSync(process.env.IBM_DB_HOME)) {
+      if (platform == 'os390'){
         IBM_DB_HOME = process.env.IBM_DB_HOME;
```

Comment:
We should probably add a check to make sure IBM_DB_HOME is set like:
```
if (!process.env.IBM_DB_HOME) {
 printMsg( "The IBM_DB_HOME environment variable must be set to the DB2 HLQ\n");
 clidriverFound = false;
} else ...
```

Reply:
Actually looks like there's logic here: https://github.com/ibmdb/node-ibm_db/blob/504b994975a5634c3da38495bd0312e55f9afb25/installer/driverInstall.js#L305

Reply:
I can confirm that it checks and prints an error if you don't have IBM_DB_HOME set on z/OS